### PR TITLE
store/mockstore: Fix that DeleteRange didn't encode keys.

### PR DESCRIPTION
**Pull request by MyonKeminta**
On MVCC store, we encode keys to memcomparable format and append ts to it. However the DeleteRange method didn't encode it.
The new test case in `mock_tikv_test.go` reveals this problem, and now it's fixed.

### CODE REVIEW

File: store/mockstore/mocktikv/mvcc_leveldb.go [hunk 1]
```diff
@@ -881,20 +881,7 @@ func (mvcc *MVCCLevelDB) BatchResolveLock(startKey, endKey []byte, txnInfos map[
...
-	iter := mvcc.db.NewIterator(&util.Range{
-		Start: startKey,
-		Limit: endKey,
-	}, nil)
-	for iter.Next() {
-		batch.Delete(iter.Key())
-	}
-
-	return mvcc.db.Write(batch, nil)
+	return mvcc.doRawDeleteRange(codec.EncodeBytes(nil, startKey), codec.EncodeBytes(nil, endKey))
```
Comment (zimulala) [inline store/mockstore/mocktikv/mvcc_leveldb.go]: Why not use `mvccEncode`?

Comment (MyonKeminta) [reply store/mockstore/mocktikv/mvcc_leveldb.go]: We don't need to append timestamp here.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**